yang-lsp/io.typefox.yang/src/main/java/io/typefox/yang/validation/YangValidator.xtend

```diff
@@ -287,6 +287,7 @@ class YangValidator extends AbstractYangValidator {
 	@Check
 	def checkPattern(Pattern it) {
 		// https://tools.ietf.org/html/rfc7950#section-9.4.5
+		// If 'it' is double quoted replace all '\\' with '\'
```

Comment:
@dhuebner could you please help how to achieve unescaping backslashes in case the pattern is double quoted